pkg/cmd/roachtest/roachtestutil/mixedversion/mixedversion_test.go

```diff
@@ -244,7 +245,7 @@ func Test_choosePreviousReleases(t *testing.T) {
 			arch:              vm.ArchAMD64,
 			numUpgrades:       3,
 			enableSkipVersion: true,
-			expectedReleases:  []string{"23.2.4", "24.1.1", "24.2.2"},
+			expectedReleases:  []string{"23.1.17", "23.2.4", "24.1.1"},
```

Comment:
x-referencing [note from MinSupport bump comment](https://github.com/cockroachdb/cockroach/pull/134340#pullrequestreview-2419388116) (for when we do this task for the next cycle), that this change is a result of the randomized sequence of versions that the test decides on with that seed.

Reply:
Right, and it's all based on the testPredecessorMapping (not the actual release data)